settings.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
         pass: process.env.API_PASSWORD || "$2a$08$s.NFdSn4Gm4d7gHErya//e6O8RO1/3f7TZ7zflXJ9jfFV0cI6jGwK"
     },
     uiPort: process.env.PORT || 1880,
+    disableEditor: process.env.DISABLE_EDITOR || false,
```

Comment:
Can we disable the editor except in the dev mode?

Reply:
why not ! but the issue is that will change the current behavior and will require a major change I guess. For upgrading the env var must be added in the running instance...

Reply:
This issue cost me a flash on the road.